Project/Project/Models/Rating.cs

```diff
@@ -12,6 +12,8 @@ public class Rating
     public int User_id { get; set; }
     public string Movie_id { get; set; }
     public double Rating_value { get; set; }
+    public string movie_title { get; set; }
+    public string movie_poster { get; set; }
```

Comment:
To follow the standard please change to capitazed example Movie_title, Movie_poster